signer/tests/integration/setup.rs

```diff
@@ -945,10 +950,21 @@ impl TestSweepSetup2 {
         }
     }
 
+    pub fn reject_withdrawal_request(&mut self) {
+        for i in 0..self.signers.keys.len() {
+            self.withdrawal_request.signer_bitmap.replace(i, true);
+        }
+    }
+
     pub async fn store_withdrawal_request(&self, db: &PgStore) {
+        let block_height = match self.sweep_block_height() {
+            Some(height) => height,
```

Comment:
I don't think we want this at the sweep block height, but maybe the deposit block height, if it's there.

Reply:
`TestSweetSetup` uses sweep block height, but `TestSweepSetup2` was using `Fake` data, so I changed `TestSweepSetup2` to use the sweep block height if available.

There's a param to include the withdrawal request:```
pub fn submit_sweep_tx(&mut self, rpc: &Client, faucet: &Faucet, with_withdrawals: bool)
```

Reply:
Oh it does? Well, just a heads up, I proposed some edits to this struct as well in https://github.com/stacks-network/sbtc/pull/1359.

Reply:
If my PR get's merged first, I don't mind updating this one because of those changes.